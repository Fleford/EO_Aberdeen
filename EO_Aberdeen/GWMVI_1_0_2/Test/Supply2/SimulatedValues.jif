jif %
StandardFile 0  2          15
Q1c       
Q2c       
Q4c       
S01_9     
S01_10    
S01_11    
S01_12    
S02_11    
S02_12    
S03_9     
S03_10    
S03_11    
S03_12    
S04_11    
S04_12    
