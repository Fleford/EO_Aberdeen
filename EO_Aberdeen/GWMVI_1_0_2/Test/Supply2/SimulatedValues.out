Q1c                     38.08947990578423    
Q2c                     72.36212894185002    
Q3c                     60.27793495385746    
Q4c                     39.36439178028365    
S01_9                   114331.6903076555    
S01_10                  120047.5014689869    
S01_11                  113237.5761780032    
S01_12                  115273.1966250864    
S02_11                  115592.8220513173    
S02_12                  118910.2444669244    
S03_9                   45271.51659776535    
S03_10                  48754.14723395879    
S03_11                  44604.61717425744    
S03_12                  46067.65416473422    
S04_11                  162265.0052202462    
S04_12                  167497.4140907284    
